example/src/main/java/io/netty/example/http2/tiles/Launcher.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.netty.example.http2.tiles;
+
+import io.netty.example.http2.tiles.http.HttpServer;
+import io.netty.example.http2.tiles.http2.Http2Server;
+
+/**
+ * Launches both Http and Http2 servers using Netty to display a set of images
+ * and simulate latency. It is a Netty version of the <a
+ * href="https://http2.golang.org/gophertiles?latency=0"> Go lang HTTP2 tiles
+ * demo</a>.
+ *
+ * Please note that you MUST use JDK 1.8 to run this example and have the
+ * associated ALPN library as described here:
+ * http://eclipse.org/jetty/documentation
+ * /current/alpn-chapter.html#alpn-versions.
+ */
+public class Launcher {
+
+    public static void main(String[] args) {
+        Http2Server http2 = new Http2Server();
+        HttpServer http = new HttpServer();
+        try {
+            http2.start();
+            System.err.println("Open your web browser and navigate to " + "http://" + Html.IP + ":" + HttpServer.PORT
+                    + "/http2?latency=0");
+            http.start().sync();
+        } catch (Exception e) {
+            e.printStackTrace();
+        } finally {
+            http2.stop();
+            http.stop();
```

Comment:
if we do https://github.com/netty/netty/pull/3620/files#r28214393 (share the EventLoops) then we can shut down 1 time and not have to worry about exceptions happening in 1 and not other or other boarder cases...